### PR TITLE
Add Feedly recipe

### PR DESCRIPTION
**Pull request by kastnerp**

### CODE REVIEW

File: uncompressed/feedly/webview.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const path = require('path');
+
+module.exports = Franz => {
+  let modal;
+  let updates = 0;
+
+  const createModal = () => {
```
Comment (kytwb) [inline uncompressed/feedly/webview.js:9]: I don't understand why a modal is being created.

**Review by kytwb — CHANGES_REQUESTED** [review]
I see there is a Franz recipe available at https://github.com/alphamyd/meetfranz-recipe-feedly, have you tried it?

File: uncompressed/feedly/webview.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  const hideModal = () => {
+    modal.querySelector('p').innerHTML = '';
+    updates--;
+    modal.classList.remove('open');
+  };
+
+  window.alert = showModal;
+
+  const getMessages = () => Franz.setBadge(updates);
```
Comment (kytwb) [inline uncompressed/feedly/webview.js:32]: I don't use Feedly so I am not sure if it is really needed to override `window.alert` with this modal logic. Isn't it possible to get the notifications count thanks to a call to `document.querySelector` like in https://github.com/alphamyd/meetfranz-recipe-feedly/blob/6b5d39962d516dd4b6cfdd74526d310f6cf321a5/webview.js#L4?

**Comment by kastnerp** [comment]
You are right, I had not tested the other one.

File: uncompressed/feedly/feedly.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    font-size: 20px;
+    font-weight: bold;
+}
+
+#franz-modal .close:hover,
+#franz-modal .close:focus {
+    color: black;
+    text-decoration: none;
+    cursor: pointer;
+}
```
Comment (kytwb) [inline uncompressed/feedly/feedly.css:41]:
```suggestion
```

**Comment by kytwb** [comment]
@kastnerp thank you for your contribution!

I merged despite the failing build as part of the refactor taking place in #292.

**Comment by kytwb** [comment]
Fixes https://github.com/getferdi/recipes/issues/233.

**Pull request merged**